src/main/java/edu/cornell/kfs/concur/aws/ConcurTokenConfig.java

```diff
@@ -0,0 +1,31 @@
+package edu.cornell.kfs.concur.aws;
+
+public class ConcurTokenConfig {
+    private String access_token;
+    private String refresh_token;
+    private String access_token_expiration_date;
```

Comment:
We were formerly using the expiration date as a string, since we were storing it in the Web Service Credentials table and that only allows for strings. However, now that your new AwsSecretService is available and can auto-parse DTOs accordingly, I agree that we could now store this value as a Date instead. I'll adjust the DTO and its related code accordingly.